scripts/source-maintenance.sh

```diff
@@ -232,10 +246,10 @@ for FILENAME in `git ls-files`; do
 	chmod 755 ${FILENAME}
 	;;
 
-    Makefile.am)
-
-    	perl -p -e 's/@([A-Z0-9_]+)@/\$($1)/g' <${FILENAME} >${FILENAME}.styled
-	mv ${FILENAME}.styled ${FILENAME}
+    *Makefile.am)
+    	perl -p -e 's/@([A-Z0-9_]+)@/\$($1)/g' <${FILENAME} | \
+			scripts/sort-makefile-am-sources.pl >${FILENAME}.styled
+		mv ${FILENAME}.styled ${FILENAME}
 	;;
```

Comment:
If this can this be done via the recently added applyPlugin API, please use that API instead.

And I would integrate the first line into the new Perl script. There is little value in keeping the two formatting changes separately (and there is some harm). And, sooner or later, we will probably want to do more than just sorting anyway.

Reply:
done both

Reply:
On second thoughts, I'm removing this; that code has been dead for a while.
It used to only apply to ${top_srcdir}/Makefile.am where it didn't match anything. Applying that to all *.am files breaks doc/manuals/Substitute.am :

```
diff --git a/doc/manuals/Substitute.am b/doc/manuals/Substitute.am
index 03e4d55e5..b10e41cdc 100644
--- a/doc/manuals/Substitute.am
+++ b/doc/manuals/Substitute.am
@@ -9,12 +9,12 @@
 ## DEFAULT_ERROR_DIR       = $(datadir)/errors

 SUBSTITUTE=sed "\
-       s%@DEFAULT_ERROR_DIR@%$(DEFAULT_ERROR_DIR)%g;\
-       s%@DEFAULT_MIME_TABLE@%$(DEFAULT_MIME_TABLE)%g;\
-       s%@DEFAULT_SSL_CRTD@%$(DEFAULT_SSL_CRTD)%g;\
-       s%@DEFAULT_SSL_DB_DIR@%$(DEFAULT_SSL_DB_DIR)%g;\
+       s%$(DEFAULT_ERROR_DIR)%$(DEFAULT_ERROR_DIR)%g;\
+       s%$(DEFAULT_MIME_TABLE)%$(DEFAULT_MIME_TABLE)%g;\
+       s%$(DEFAULT_SSL_CRTD)%$(DEFAULT_SSL_CRTD)%g;\
+       s%$(DEFAULT_SSL_DB_DIR)%$(DEFAULT_SSL_DB_DIR)%g;\
        s%@""PACKAGE_STRING""@%$(PACKAGE_STRING)%g;\
-       s%@SYSCONFDIR@%$(sysconfdir)%g;\
+       s%$(SYSCONFDIR)%$(sysconfdir)%g;\
        "

 ## Example of how to substitute:
```

Reply:
I am resolving this partially addressed change request. I will submit a fresh one to avoid bundling two problems into one change request.